uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerAuthenticator.java

```diff
@@ -32,24 +32,21 @@
 import androidx.test.uiautomator.UiObjectNotFoundException;
 import androidx.test.uiautomator.UiSelector;
 
-import com.microsoft.identity.client.ui.automation.app.App;
 import com.microsoft.identity.client.ui.automation.installer.PlayStore;
-import com.microsoft.identity.client.ui.automation.interaction.AadPromptHandler;
 import com.microsoft.identity.client.ui.automation.interaction.PromptHandlerParameters;
 import com.microsoft.identity.client.ui.automation.interaction.PromptParameter;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.AadPromptHandler;
 import com.microsoft.identity.client.ui.automation.utils.CommonUtils;
 import com.microsoft.identity.client.ui.automation.utils.UiAutomatorUtils;
 
 import org.junit.Assert;
 
 import lombok.Getter;
 
-import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
 import static com.microsoft.identity.client.ui.automation.utils.CommonUtils.FIND_UI_ELEMENT_TIMEOUT;
-import static com.microsoft.identity.client.ui.automation.utils.CommonUtils.getResourceId;
 
 @Getter
-public class BrokerAuthenticator extends App implements ITestBroker {
+public class BrokerAuthenticator extends AbstractTestBroker implements ITestBroker {
```

Comment:
nit: suggest renaming to BrokerMicrosoftAuthenticator to make it more clear (Authenticator is also used as a class name inside Broker)

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/944/commits/5e4381ba6244eb59690bf94ba865de1e8dc17671